### PR TITLE
New Change from ataur

### PR DESCRIPTION
**Pull request by ataurnyc**

### CODE REVIEW

File: src/test/java/com/xyz/MavenExample/GroupTestNG.java [hunk 1]
```diff
@@ -26,5 +26,6 @@ public void a3_search() throws InterruptedException {
 			driver.findElement(By.name("q")).sendKeys("NeW York");
 			driver.findElement(By.name("q")).submit();
 			Thread.sleep(3000);
+			driver.close();
```
Comment (ImMayankGupta) [inline src/test/java/com/xyz/MavenExample/GroupTestNG.java:29]: Why did you close the browser?

Comment (MahadiNaim) [reply src/test/java/com/xyz/MavenExample/GroupTestNG.java:29]: to check close tab

**Review by ImMayankGupta — APPROVED** [review]

**Comment by mustak87** [comment]
I accepted request.

File: src/test/java/com/xyz/MavenExample/TestNG_BASMCMG.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	
+	
+	@Test(description ="Testcase One")
+	public void m4(){
+		System.out.println("Hello");
+	}
+	
+	@Test(description ="Testcase Two")
+	public void m5(){
+		System.out.println("World");
```
Comment (MahadiNaim) [inline src/test/java/com/xyz/MavenExample/TestNG_BASMCMG.java:34]: why did you print?

File: src/test/java/com/xyz/MavenExample/GroupTestNG.java (hunk 1 above)
Comment (ataurnyc) [reply src/test/java/com/xyz/MavenExample/GroupTestNG.java:29]: Testing

File: src/test/java/com/xyz/MavenExample/TestNG_BASMCMG.java (hunk 2 above)
Comment (ataurnyc) [reply src/test/java/com/xyz/MavenExample/TestNG_BASMCMG.java:34]: Testing

**Review by MahadiNaim — APPROVED** [review]

**Pull request merged**